tests/_utils/run_lightning_local

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+#
+# Copyright 2019 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+TEST_DIR=/tmp/lightning_test_result
+
+echo "[$(date)] <<<<<< RUNNING TEST FOR: tests/$TEST_NAME $@ >>>>>>" >> "$TEST_DIR/lightning-local.log"
+bin/tidb-lightning.test -test.coverprofile="$TEST_DIR/cov.$TEST_NAME.$$.out" DEVEL \
+    --ca "$TEST_DIR/tls/ca.pem" \
+    --cert "$TEST_DIR/tls/lightning.pem" \
+    --key "$TEST_DIR/tls/lightning.key" \
+    --log-file "$TEST_DIR/lightning.log" \
+    --tidb-port 4000 \
+    --pd-urls '127.0.0.1:2379' \
+    --config "tests/$TEST_NAME/config.toml" \
+    -d "tests/$TEST_NAME/data" \
+    --backend "local" \
+    --sorted-kv-dir "tests/$TEST_NAME/sorted" \
```

Comment:
no you should not write temporary data into the source directory. just keep using `$TEST_DIR/.../`.